src/pages/AdminManageDarCollections.js

```diff
@@ -26,7 +26,7 @@ export default function AdminManageDarCollections() {
   useEffect(() => {
     const init = async() => {
       try {
-        const collectionsResp = await Collections.getCollectionsByRoleName('admin');
+        const collectionsResp = await Collections.getCollectionSummariesByRoleName('admin');
```

Comment:
```suggestion
        const collectionsResp = await Collections.getCollectionSummariesByRoleName(USER_ROLES.admin);
```

nit: use existing user role constant